provision.sh

```diff
@@ -190,20 +200,26 @@ install_golang() {
 
 install_nodejs() {
   echo ">>> Installing NodeJS"
-  curl -fsSL https://deb.nodesource.com/setup_16.x | bash -
-  apt-get -y install nodejs
+  #curl -fsSL https://deb.nodesource.com/setup_16.x | bash -
+  #apt-get -y install nodejs
+  curl -sL https://nsolid-deb.nodesource.com/nsolid_setup_4.x | sudo bash -
+  apt-get -y install nsolid-gallium nsolid-console
 }
 
 install_gcloud_cli() {
   echo ">>> Installing the Google Cloud CLI"
-  snap remove google-cloud-sdk
-  snap install google-cloud-cli --classic
+  #snap remove google-cloud-sdk
+  #snap install google-cloud-cli --classic
+  curl -O https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-cli-445.0.0-linux-x86_64.tar.gz
+  tar -xf google-cloud-cli-445.0.0-linux-x86_64.tar.gz
+  ./google-cloud-sdk/install.sh -q --rc-path ~/.bashrc
+  source ~/.bashrc
 }
 
 install_cf_tools() {
```

Comment:
The `echo ">>> Installing the Cloud Foundry CLI"` line is useful so that it would be visible what is going on while waiting for the provisioning to complete